x/validatorreward/keeper/distribute.go

```diff
@@ -48,5 +48,8 @@
 
 	k.SetLastDistributeTime(ctx, ctx.BlockHeader().Time)
 
-	k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, types.ValidatorRewardCollectorName, rewardAmount)
+	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, types.ValidatorRewardCollectorName, rewardAmount)
+	if err != nil {
+		panic(err)
```

Comment:
It's ok to panic here since `SendCoinsFromModuleToModule` is very basic action, and amount to send already check with current balance of module account